Chromium-XP_49_0_2623_113_VS2013_SSE3.txt

```diff
@@ -0,0 +1,10 @@
+[Section]
+Name = Chromium-XP
+Version = 49.0.2623.113 VS2013 SSE3 1st Release
+License = BSD 3-Clause License
+Description = Prebundled with Pepperflash 21.0.0.213. 489/555 in html5test.com. Currently needs --no-sandbox command line option to run on ROS.
```

Comment:
I think it would be better "to run on ReactOS." explicitly, but also just "to run." should probably be intuitive enough.

Reply:
Ok, I will use "to run".